app/services/the_tvdb_service/row.rb

```diff
@@ -0,0 +1,89 @@
+class TheTvdbService
+  class Row
+    attr_reader :media_id, :media_type, :data, :episode_id, :tvdb_series_id
+
+    def initialize(media_id, data, tvdb_series_id)
+      @media_id = media_id
+      @data = data
+      @episode_id = data['id']
+      @tvdb_series_id = tvdb_series_id
+      @media_type = 'Anime'
+    end
+
+    def update_episode
+      return unless episode_number.present?
+
+      # All these methods will be modifying the episode object in-place
+      %i[
+        season_number relative_number synopsis
+        thumbnail airdate titles canonical_title
+      ].each { |k| public_send(k) }
+
+      # to get the imdbId we would have to call the /episodes/{episode_number} endpoint
+      # instead of the series/episdoes.
+      # TODO: implement getting imdbId (which is an episode id)
+      # once we can store mappings for an episode.
```

Comment:
We can now store mappings for an episode

`Mapping.create(item: episode, external_site: 'imdb/episodes', external_id: imdbId)`

Reply:
for `item:` are we taking the kitsu episode_id?

Reply:
It's an association my dude